scripts/tests/ota_test.sh

```diff
@@ -15,30 +18,28 @@ OTA_REQUESTOR_FOLDER="out/ota_requestor_debug"
 CHIP_TOOL_APP="chip-tool"
 CHIP_TOOL_FOLDER="out"
 
-killall -e "$OTA_PROVIDER_APP" "$OTA_REQUESTOR_APP"
+killall -INT -e "$OTA_PROVIDER_APP" "$OTA_REQUESTOR_APP"
 rm -f "$FIRMWARE_OTA" "$FIRMWARE_BIN" "$OTA_DOWNLOAD_PATH"
 
-set -e
-
 scripts/examples/gn_build_example.sh examples/chip-tool "$CHIP_TOOL_FOLDER"
 scripts/examples/gn_build_example.sh examples/ota-requestor-app/linux "$OTA_REQUESTOR_FOLDER" chip_config_network_layer_ble=false
 scripts/examples/gn_build_example.sh examples/ota-provider-app/linux "$OTA_PROVIDER_FOLDER" chip_config_network_layer_ble=false
 
 echo "Test" >"$FIRMWARE_BIN"
 
-rm /tmp/chip_*
+rm -r /tmp/chip_*
```

Comment:
Add "-f" as well, to ignore non-existent fails and not fail.

Reply:
will do